ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionlayer/PayloadBuildingAttributes.java

```diff
@@ -91,12 +97,19 @@ public boolean equals(final Object o) {
         && Objects.equals(prevRandao, that.prevRandao)
         && Objects.equals(feeRecipient, that.feeRecipient)
         && Objects.equals(validatorRegistration, that.validatorRegistration)
-        && Objects.equals(maybeWithdrawals, that.maybeWithdrawals);
+        && Objects.equals(maybeWithdrawals, that.maybeWithdrawals)
+        && Objects.equals(parentBeaconBlockRoot, that.parentBeaconBlockRoot);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(timestamp, prevRandao, feeRecipient, validatorRegistration);
+    return Objects.hash(
```

Comment:
This was missed before I suppose but can add blockSlot to equals,hashCode and toString

Reply:
ye noticed that too.. for some reason I ended up not adding it